packages/gatsby-plugin-netlify-cms/src/gatsby-node.js

```diff
@@ -29,7 +29,27 @@ function deepMap(obj, fn) {
   return obj
 }
 
-function replaceRule(value) {
+const cssTests = []
+
+function isCssRule({ test }) {
+  return test instanceof RegExp && cssTests.includes(test.toString())
+}
+
+function replaceStyleLoader(rule) {
+  if (rule.loader.includes(`style-loader`)) {
```

Comment:
This function is meant to be used within `replaceRule` after it passes the stage check. I wonder if it makes sense for future developers to put the stage check in this function, or inline this function inside of `replaceRule`. I suppose the potential risk I see is that `replaceStyleLoader` seems generic and reusable, but it's intended to only apply to develop.

Reply:
Thanks for looking into this. Totally makes sense. I've moved it into `replaceRule()`.